README.md

```diff
@@ -22,14 +22,28 @@ We'd like to welcome you to Hackin' Omics 2022! We hope you are as excited as we
 - [ ] Fork this repository.
 - [ ] Review the [CODE_OF_CONDUCT](CODE_OF_CONDUCT.md)
 - [ ] Review some of the helpful resources we listed in the resources [README](https://github.com/u-brite/team-leaders-2022/blob/main/resources/README.md).
-- [ ] Create a GitHub repository for your hackathon team on our **U-BRITE organization** (not on your GitHub profile).
+    - [ ] Review what it takes to be a good team leader.
+    - [ ] Ensure your team members are on slack and be sure to introduce yourself and the project.
+    - [ ] Remember to discuss authorship and responsibilities up front as some projects may result in publications.
+- [ ] Create a GitHub repository on our **U-BRITE organization** (not on your GitHub profile).
     - [ ] Use the ["team-repo"](https://github.com/u-brite/team-repo-template) template and update with your team's information.
+    - [ ] Your repository should be PUBLIC.
     - [ ] Refine your project proposal based on the feedback given (if there was feedback).
-- [ ] Attend workshops to become ready for the hackathon.
-    - [ ] Computing Resources Workshop: An Introduction to Jetstream2, HPC, and Git/GitHub | Register at https://tinyurl.com/mubxwc8f | July 27 at 2PM
-    - [ ] U-BRITE Bootcamp: Learn how to use PAGER & SEAS for multi-omics downstream analysis by Dr. Jake Chen | Register at https://tinyurl.com/yxx23fph | July 29 at 10AM
+    - [ ] Add your team members to the [U-BRITE organization](https://github.com/orgs/u-brite/teams/hackin-omics/members) if they have a GitHub account or have them create an account.
+- [ ] Attend workshops or watch recordings to become ready for the hackathon.
+    - [ ] [Computing Resources Workshop: An Introduction to Jetstream2, HPC, and Git/GitHub](https://uab.box.com/s/ddsxtndjci74uob55xil6qwcrp6ksjjl) by [Dr. Blake Joyce](https://github.com/bjoyce3)
+    - [ ] [U-BRITE Bootcamp: Learn how to use PAGER & SEAS for multi-omics downstream analysis]() [by Dr. Jake Chen](https://github.com/jakechen1)
```

Comment:
add this link to the bootcamp since we don't have a video - https://github.com/u-brite/u-brite-bootcamp